### PR TITLE
[FCE-236] Create basic README

### PR DESCRIPTION
**Pull request by mironiasty**
## Description

This file adds main README into repository. It has basic description what all web SDKs do. There might be old links to docs left, but I'm aware they are broken. We have to work on updated docs after we finish polishing these SDKs

I also remove old references to membrane in other files. And updated README for packages, to have only information that are now relevant.

## Motivation and Context

It would be nice to have at least basic README in repo.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

### CODE REVIEW

File: examples/react-client/readme.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 List of examples:
 
-- [minimal-react](https://github.com/fishjam-dev/react-client-sdk/tree/main/examples/minimal-react) - minimal
-  example of usage of Fishjam React client:
+- [minimal-react](./minimal-react/) - minimal example of usage of Fishjam React Client:
 
   - TypeScript with React Client
   - connect to Fishjam server
```
Comment (karkakol) [inline examples/react-client/readme.md:8]: Fishjam Server ?

Comment (karkakol) [reply examples/react-client/readme.md:8]: Fishjam Cloud?

File: examples/react-client/use-camera-and-microphone/README.md [hunk 2]
```diff
@@ -4,19 +4,19 @@
 
 Follow these steps to start and run your project:
 
-1. Install Dependencies: Use npm to install the project dependencies:
+1. Install Dependencies: Use yarn to install the dependencies in root directory:
 
 ```shell
 npm install
 ```
```
Comment (karkakol) [inline examples/react-client/use-camera-and-microphone/README.md:11]: Use yarn to install and then command uses npm? 
It should be consistent with main readme, so one command for yarn and one for npm.

File: packages/ts-client/README.md [hunk 3]
```diff
@@ -117,31 +122,13 @@ async function startScreenSharing(webrtc: WebRTCEndpoint) {
...
+For more examples, see the [examples](../../examples/ts-client/) folder.
 
 ## Copyright and License
 
 Copyright 2024,
-[Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=fishjam)
+[Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=fishjam-ts)
 
-[![Software Mansion](https://logo.swmansion.com/logo?color=white&variant=desktop&width=200&tag=membrane-github)](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=fishjam)
+[![Software Mansion](https://logo.swmansion.com/logo?color=white&variant=desktop&width=200&tag=fishjam-github)](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=fishjam-ts)
```
Comment (karkakol) [inline packages/ts-client/README.md:132]: Double logo image and link?
One should be enough.

File: packages/ts-client/package.json [hunk 4]
```diff
@@ -3,13 +3,12 @@
   "version": "0.5.0",
   "description": "Typescript client library for Fishjam",
   "license": "Apache-2.0",
-  "author": "Software Mansion (https://swmansion.com)",
+  "author": "Fishjam Cloud Team",
```
Comment (karkakol) [inline packages/ts-client/package.json:6]: mem

File: examples/react-client/readme.md (hunk 1 above)
Comment (mironiasty) [reply examples/react-client/readme.md:8]: Updated

File: examples/react-client/use-camera-and-microphone/README.md (hunk 2 above)
Comment (mironiasty) [reply examples/react-client/use-camera-and-microphone/README.md:11]: I missed that line, fixed

**Review by karkakol — APPROVED** [review]
🥔

**Pull request merged**